ocs_ci/helpers/disruption_helpers.py

```diff
@@ -84,12 +106,17 @@ def select_daemon(self, node_name=None):
             node_name (str): Name of node in which the resource daemon has
                 to be selected.
         """
+        kubeconfig_parameter = (
```

Comment:
I think we can add the var `kubeconfig_parameter` to the class. (e.g `self.kubeconfig_parameter`)

Reply:
Added a function to return the parameter. This will help to avoid adding an additional variable in the function set_resource.